### ISSUE TITLE
False-positives

### ISSUE THREAD

**Issue by EvgenyOrekhov** [amilajack/eslint-plugin-compat#186]
I believe these are false-positives:
```
document.getElementsByTagName() is not supported in Safari 11.1, Opera 57, Edge 17
document.documentElement() is not supported in Safari 11.1, Opera 57, Edge 17
window.history() is not supported in Safari 11.1, Opera 57, Edge 17
```

eslint-plugin-compat version: 3.0.0

**Comment by vikr01** [comment]
I believe the analyses for supported versions are created by [browserslist](https://github.com/browserslist/browserslist), so you may want to file an issue there instead.

**Comment by amilajack** [comment]
The versions are generated by browsers list but the compat records are from [ast-metadata-inferer](https://github.com/amilajack/ast-metadata-inferer) and [mdn-browser-compat-data]

**Comment by amilajack** [comment]
@EvgenyOrekhov I fixed this in the `@next` tag. Let me know if it fixes this bug:
```
npm install eslint-plugin-compat@next
```

**Comment by EvgenyOrekhov** [comment]
@amilajack Yes, it fixes the bug, thanks!

**Comment by amilajack** [comment]
🎉
